tests/integration/generic_test.go

```diff
@@ -28,75 +27,79 @@ var _ = Describe("odo generic", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("Check the help usage for odo", func() {
+	When("running odo --help", func() {
+		It("retuns full help contents including usage, examples, commands, utility commands, component shortcuts, and flags sections", func() {
+			output := helper.Cmd("odo", "--help").ShouldPass().Out()
```

Comment:
We only use `When` when there is some action to be performed before testing, in this case, it is not necessary to use `When`, you can simply make it work with `It`.
```suggestion
		It("should return full help contents including usage, examples, commands, utility commands, component shortcuts, and flags sections when running odo --help", func() {
			output := helper.Cmd("odo", "--help").ShouldPass().Out()
```

Or, if you want to use `When`, you can do something like the following:
```suggestion
	When("running odo --help", func() {
		var output string
		BeforeEach(func() {
			output = helper.Cmd("odo", "--help").ShouldPass().Out()
		})
		It("should return full help contents including usage, examples, commands, utility commands, component shortcuts, and flags sections", func() {
```